### PR TITLE
feat(icons): added and updated new icons

### LINKED ISSUE #2088: 230626 - Icon Updates
We have some icon updates:

**New icons**
- Phone (16)
- Package (16)
- Discount (24, 16)
- Promotion (24, 16)
- Arrows 3D filled (64) _-colored_
- Search Similar (24)

**Updated artwork**
- Search Similar (20, 16)
- Phone (24)
- Package (24)
- Pause filled (64) _-colored_
- Wrench (24, 16)

[230725_Icon_Updates.zip](https://github.com/eBay/skin/files/12166019/230725_Icon_Updates.zip)

### PR DESCRIPTION
**Pull request by ArtBlue**
Fixes #2088 

- [X] This PR contains CSS changes
- [ ] This PR does not contain CSS changes

## Description
Added and updated new icons.

## Checklist

- [X] I verify the build is in a non-broken state
- [X] I verify all changes are within scope of the linked issue

- [X] I regenerated all CSS files under dist folder
- [X] I tested the UI in all supported browsers
- [X] I did a visual regression check of the components impacted by doing a Percy build and approved the build
- [X] I tested the UI in dark mode and RTL mode
- [X] I added/updated/removed Storybook coverage as appropriate

### CODE REVIEW

File: dist/alert-dialog/alert-dialog.css [hunk 1]
```diff
@@ -2,7 +2,6 @@
   --dialog-scrim-color-hide: rgba(17, 24, 32, 0);
   --dialog-scrim-color-show: rgba(17, 24, 32, 0.7);
   --dialog-lightbox-max-width: 616px;
-  --dialog-lightbox-wide-max-width: 896px;
```
Comment (agliga) [inline dist/alert-dialog/alert-dialog.css:5]: Might need to pull changes from 16.6.0

**Review by agliga — APPROVED** [review]
LGTM
One minor comment: it seems like we need to update the references from 16.6.0. Try rebasing before merging.

File: src/less/lightbox-dialog/stories/lightbox-dialog.stories.js [hunk 2]
```diff
@@ -279,6 +279,29 @@ export const wide = () => `
 </div>
 `;
 
+export const narrow = () => `
```
Comment (ArtBlue) [inline src/less/lightbox-dialog/stories/lightbox-dialog.stories.js]: @agliga , I'm not sure why this diff showed up. I pulled in `origin 16.6.0`.

File: dist/alert-dialog/alert-dialog.css (hunk 1 above)
Comment (ArtBlue) [reply dist/alert-dialog/alert-dialog.css]: Done.

**Pull request merged**